README.md

```diff
@@ -1,3 +1,17 @@
 Data Together
 ========
 > Community Patterns for the Decentralized Web
+
+We are engaging in a collaborative process to more clearly describe Data Together so that the initiative can be better understood by newcomers. This process began during an in-person meeting in Boston on August 29, 2017 with @dcwalk, @flyingzumwalt, @jeffreyliu, @mhucka, @ebarry, @b5, @jschell42, @titaniumbones, @rgardaphe. We are starting with time-tested patterns from non-profits, collaboratives, and companies: Mission, Vision, Values. Please click on the links below to see and comment on the draft text:    
+
+* **[Mission](/mission.md)** -- our purpose for existing, our charge, our goal, what this initiative does. (1 succinct sentence only)
```

Comment:
Given that everything is so concise, I wonder if we could pull in MVV here, and have the links out as you've structured? 

(also imagining that these will go on the website!)

Reply:
sure! Let's do that. 

How about leaving the template below them? My perception is that people who don't normally hang out with non-profits are less familiar with type of structured text.